File: PSoft.Libraryd/PSoft.Libraryd.Domain/DTOs/AlquilerDTO.cs
using System;

namespace PSoft.Libraryd.Domain.DTOs
{
    public class AlquilerDTO
    {
        public int Cliente { get; set; }
        public string ISBN { get; set; }
        public DateTime? FechaReserva { get; set; }
        public DateTime? FechaAlquiler { get; set; }
    }
}

File: PSoft.LibrarydClient/public/js/DTOs/AlquilerDTO.js
export class AlquilerDTO {
  constructor(clienteId, isbn, fechaAlquiler) {
    this.cliente = clienteId;
    this.isbn = isbn;
    this.fechaAlquiler = fechaAlquiler;
  }
}

export class ReservaDTO {
  constructor(clienteId, isbn, fechaReserva) {
    this.cliente = clienteId;
    this.isbn = isbn;
    this.fechaReserva = fechaReserva;
  }
}
